graphviz2drawio/mx/MxGraph.py

```
@@ -8,6 +8,7 @@
 from graphviz2drawio.mx.Edge import Edge
 from graphviz2drawio.mx.Node import Node
 from graphviz2drawio.mx.Styles import Styles
+from graphviz2drawio.models import SVG
```

Ruff (0.8.0)

`graphviz2drawio.models.SVG` imported but unused

Remove unused import: `graphviz2drawio.models.SVG`

(F401)